storage/aptosdb/src/state_store/mod.rs

```diff
@@ -649,6 +649,28 @@ impl StateStore {
             expected_root_hash,
         )?))
     }
+
+    /// Prune the stale state value schema generated between a range of version in (begin, end]
```

Comment:
"[begin, end)"

Reply:
It is confusing because stale_since_version will be `end` inclusive.